cider/exec/nextgen/parsers/RelAlgEUParser.cpp

```diff
@@ -70,19 +71,18 @@ class InputAnalyzer {
       }
     }
 
-    for (size_t i = 0; i < input_exprs_.size(); ++i) {
-      if (nullptr == input_exprs_[i]) {
-        LOG(FATAL) << "Input column expr missed. Column descriptor = "
-                   << input_desc_[i].toString();
-      }
-    }
+    input_exprs_.erase(
+        std::remove_if(input_exprs_.begin(),
+                       input_exprs_.end(),
+                       [](const ExprPtr& input_expr) { return input_expr == nullptr; }),
+        input_exprs_.end());
```

Comment:
why?

Reply:
not all column will be used in the table

Reply:
When `input_expr` is null and not used? Could you show me an example?

Reply:
for example, only select one column, but the table has two columns. it is the eu parser code.

Reply:
got it